src/main/java/spoon/Metamodel.java

```diff
@@ -146,6 +146,11 @@ private Metamodel() { }
 		result.add(factory.Type().get(spoon.reflect.reference.CtUnboundVariableReference.class));
 		result.add(factory.Type().get(spoon.reflect.reference.CtVariableReference.class));
 		result.add(factory.Type().get(spoon.reflect.reference.CtWildcardReference.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtModule.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtModuleRequirement.class));
```

Comment:
CtModuleProvidedService should be called  CtProvidedService

add CtUsedService